### ISSUE TITLE
Add new column in QC to allow ranking of segmentations

### ISSUE THREAD

**Issue by plbenveniste** [spinalcordtoolbox/spinalcordtoolbox#4758]
In the context of ms lesion segmentation, we want to have the possibility of ranking lesion segmentation. This means, assigning each QC line a value between 1 and 5. 

To do so, we want to create a new column where these values would be stored. This column will be named "QC ranking". The values would not be entered, but the person QCing would have to select some of the possible values (similarly to selecting "pass", "fail"...). The possible values will be: None, 1, 2, 3, 4 and 5. 

**Alternatives:** 

We had discussed adding these values to the existing QC column, where we store pass or fail. However, because in this context we don't want people filling "fail", I would suggest creating a separate column which could only store 1 to 5. 

Other discussion: Should we limit the values to None, 1, 2 and 3 (instead of None, 1, 2, 3, 4 and 5). 

Other discussion: we can also discuss the name of this new column.

**Help wanted**

I struggle to understand the organization of the QC construction with the multiple coding language involved. For that, I would appreciate if some people could guide me in what should be done. 

Tagging @jcohenadad @joshuacwnewton @mguaypaq for additional feedback on this feature request

**Comment by jcohenadad** [comment]
> Other discussion: Should we limit the values to None, 1, 2 and 3 (instead of None, 1, 2, 3, 4 and 5).

I would allow None to 9, and tell people to stick to 1-3 if we want a three-way ranking. That way, if one day we need to rank 4 cases, or 8 cases, it would be possible.

> Other discussion: we can also discuss the name of this new column.

How about `Grade`, which could either be a ranking (ie: 1 is best) or a score (ie: max is best)

@joshuacwnewton @mguaypaq since you are more familiar with the QC report, do you think you could help implement that additional column? Thanks!

**Comment by plbenveniste** [comment]
As I was saying in today's meeting, this is becoming relatively urgent. Could we discuss this soon ?

**Comment by joshuacwnewton** [comment]
Yes, absolutely. I'll start on it today. Apologies for the delay, and thank you for letting us know of this task's urgency.
